apps/examples/testcase/le_tc/kernel/tc_libc_unistd.c

```diff
@@ -241,6 +242,10 @@ static void tc_libc_unistd_pipe(void)
 
 	sem_wait(&pipe_sem);
 
+	/* wait for child to exit, and store child's exit status */
+	ret_chk = wait(&status);
```

Comment:
How about removing sem_wait(&pipe_sem) and add waitpid pid[0], pid[1]?
This task may not be posted from sem_post.
And is status variable necessary?